libs/cardano-ledger-test/src/Test/Cardano/Ledger/BaseTypes.hs

```diff
@@ -39,66 +40,61 @@ boundedRationalTests ::
 boundedRationalTests badJSONValues =
   testGroup
     (showsTypeRep (typeRep (Proxy :: Proxy a)) "")
-    [ testGroup
-        "Rational roundtrip"
-        [ testProperty "(boundRational . unboundRational)" $ \(bi :: a) ->
-            Just bi === boundRational (unboundRational bi),
-          testProperty "(unboundRational . boundRational)" $
-            forAll genValid $ \r ->
-              maybe
-                (property True)
-                ((r ===) . unboundRational)
-                (boundRational r :: Maybe a),
-          testProperty "bounding produces valid values within bounds" $
-            forAll genValid $ \r ->
-              case boundRational r of
-                Nothing -> property True
-                Just (br :: a) ->
-                  conjoin
-                    [ minBound <= br,
-                      br <= maxBound,
-                      unboundRational (minBound :: a) <= r,
-                      r <= unboundRational (maxBound :: a)
-                    ]
-        ],
-      testGroup "JSON" $
-        [ testProperty "ToJSON/FromJSON roundtrip up to an epsilon" $ \(br :: a) ->
-            within 500000 $
-              case eitherDecode (encode br) of
-                Left err -> error err
-                Right (br' :: a) ->
-                  epsilonMaybeEq 1e-18 (unboundRational br) (unboundRational br'),
-          testProperty "Roundtrip to Scientific and back up to an epsilon" $ \ui ->
-            within 500000 $
-              case fromRationalRepetendLimited 20 (unboundRational ui) of
-                Left (s, r) -> Just ui === boundRational (toRational s + r)
-                Right (s, Nothing) ->
-                  classify
-                    True
-                    "no-repeat digits"
-                    (Right ui === boundedFromJSON (encode s))
-                Right (s, Just r) ->
-                  Just ui === boundRational (toRationalRepetend s r),
-          testProperty "Roundtrip from valid Scientific and back exactly" $
-            within 500000 $
-              forAll genValid $ \(s :: Scientific) ->
-                case eitherDecode (encode s) of
-                  Right (ui :: a) -> s === fromRational (unboundRational ui)
-                  Left _ -> property True,
-          localOption (mkTimeout 500000) $
-            testCase "Check divergence" $
-              expectLeft (boundedFromJSON "10e1234567893456")
-        ]
-          ++ [ testCase testName $ expectLeft $ boundedFromJSON invalidInput
-               | (testName, invalidInput) <- badJSONValues
-             ],
-      testProperty "CBOR roundtrip" $ \(br :: a) ->
-        either (error . show) (br ==) $ Binary.decodeFull (Binary.serialize br),
-      testProperty "CBOR BoundedRational roundtrip" $ \(br :: a) ->
```

Comment:
This `CBOR BoundedRational roundtrip`  test removed on purpose?

Reply:
Yes, `boundedRationalToCBOR`/`boundedRationalFromCBOR` functions have been removed